cmd/setup-etcd-environment/run.go

```diff
@@ -86,9 +105,54 @@ func runRunCmd(cmd *cobra.Command, args []string) error {
 		out = f
 	}
 
-	if err := writeEnvironmentFile(map[string]string{
-		"DISCOVERY_SRV": runOpts.discoverySRV,
-	}, out, true); err != nil {
+	export := make(map[string]string)
+	etcdName := os.Getenv("ETCD_NAME")
+	if etcdName == "" {
+		return fmt.Errorf("environment variable ETCD_NAME has no value")
+	}
+	var e EtcdScaling
+	if runOpts.pivot {
+		clientConfig, err := rest.InClusterConfig()
```

Comment:
Does this mean that it requires k8s networking to be up to talk to apiservers... That would be difficult in recovery scenarios

Reply:
This functionality is part of cluster bootstrap and relies on the operator so networking would be assumed. DR would require standing up a single node etcd cluster then scaling additional members via operator.

Reply:
But I will consider how to handle this for that single node. thanks.